src/content/get-started/cloud-provider-guides/gke.mdx

```diff
@@ -137,4 +130,4 @@ After a successful installation, you can access your Okteto instance at `https:/
 
 > **Important**: The default installation is not recommended for production use. We highly advise following the steps outlined in the [Admin Dashboard](admin/dashboard.mdx#installation) before giving your team access to your Okteto instance.
 
-The default installation is all that's required to follow our [getting started](get-started/install-okteto-cli.mdx) guide.
+[Next](get-started/install-okteto-cli.mdx) we're going to install and configure the Okteto CLI with your Okteto instance.
```

Comment:
I'm not sure how easy to find those links are. They are the last sentence of the screen and `Next` or `getting started` are short words to see them easily. I think links are good to take the user to other relevant sections, but if we want to have a flow/guide of different steps, links in a paragraph are hard to find.

Should we consider to add a flow with X steps with the typical footer to next or previous section? In my mind, it would be something like:
* `get-started/overview` would have links to the public cloud providers guides.
* Each public provider guide would have a previous button to `get-started/overview` and a next button to "install the CLI"
* "Install CLI" would have a previous step to "choose the public cloud provider" and a next button to "deploy your first dev environment".
* "Deploy your first dev environment" would have a previous link to "install the CLI" and a next page to "What's next?" where we can have an index to other sections in the docs like: complete your installation (bring your own certificate, configure the auth provider, etc..), features discoverability (preview envs, install GitHub integration, catalog, private registries, etc...)

Reply:
I agree with Nacho here. I think the hit area for this link should be larger and very obvious as the next step. Maybe even a button using `pagination_next` ?

Reply:
+1 I am going to merge as it is a create a dedicated PR for this